.travis.yml

```diff
@@ -44,7 +44,7 @@ jobs:
 
     - stage: test
       python: 3.6
-      install: pip install tox
+      install: pip install tox gunicorn uvicorn starlette
```

Comment:
Do these need to be here and in tox.ini?

Reply:
I guess you are right. In the past, tox did not download dependencies. Let me quickly remove it from there